src/book/dto/create-book.dto.ts

```diff
@@ -50,8 +50,8 @@ export class CreateBookDto {
   @IsDefined({ message: "Categories must not be empty!" })
   categories!: string[];
 
-  @ApiProperty({ description: "Date of the book release", default: new Date(Date.now()) })
-  @IsDate({ message: "Realease Date must be a valid Date type!" })
+  @ApiProperty({ description: "Date of the book release", default: 1519211809934 })
```

Comment:
Why in stead of date.now() that string?

Reply:
It doesn't really matter, mainly I changed, it's the value that will be shown in the swagger API docs, I changed the `new Date` as I decided to just save an epoch date, which is a 10 digit number, I could have left the `Date.now()` though